inst/include/Rcpp/Nullable.h

```diff
@@ -76,9 +76,8 @@ namespace Rcpp {
          *
          * @throw 'not initialized' if object has not been set
          */
-        inline operator SEXP() {
-            checkIfSet();
```

Comment:
You're right, it's an unrelated change. I included it, because it affects code in close proximity to the code that I was touching anyway. The reason for that change: in the unlikely event that the implementation of get() changes, you will also want to change the implementation of operator SEXP(), because they seem synonymous.

That said, I'm totally fine with excluding that particular change from this PR, if you prefer that.